### DISCUSSION TITLE
Small mistake on Modbus TCP

### DISCUSSION THREAD

**Discussion by Imanol82** [kellerza/sunsynk#327] [Q&A] [answered]
Hello,

I have noticed that there is a small error in one of the sensors in the Modbus_TCP.

sensor.battery_charge_limit_current and sensor.battery_discharge_limit_current should be 133A and are displayed as 1.33A

Thanks for your work!

**Comment by kellerza** [comment]
Hi, thanks for reporting.
These depend on the definitions. Which ones are you using? 1ph, 3ph lv/hv

**Comment by Imanol82** [comment, marked as answer]
1ph.

![image](https://github.com/user-attachments/assets/ec8003f6-9ce2-4b2f-8ac1-46c8247d6b4d)

Thank you very much.
